src/olympia/reviewers/utils.py

```diff
@@ -622,7 +622,6 @@ def reviewer_reply(self):
 
     def process_comment(self):
         if self.version:
-            kw = {'has_reviewer_comment': True}
```

Comment:
@eviljeff can you please guide me what to do here i think it will fail after removing from here

Reply:
You can collapse it down to a single if and `.update()` call.
```python
update_reviewed = (
    self.version and
    self.version.channel ==  amo.RELEASE_CHANNEL_UNLISTED and
    not self.version.reviewed)
if update_reviewed:
    self.version.update(reviewed=datetime.now())
```